src/cmd/services/m3query/config/config.go

```diff
@@ -122,6 +122,9 @@ type Configuration struct {
 
 	// ResultOptions are the results options for query.
 	ResultOptions ResultOptions `yaml:"resultOptions"`
+
+	// Cache configurations.
```

Comment:
nit: add a comment explaining deprecation and rename var. something like:

```
// Cache configurations.
// 
// Deprecated: cache configurations are no longer supported. Remove from file. 
DeprecatedCache CacheConfiguration `yaml:"cache"`
```

Reply:
Okay fair enough - i was gonna leave in case we wanted to use this field again, but we can just change the comment if we do.